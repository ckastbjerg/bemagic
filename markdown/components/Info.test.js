/* eslint-env jest */
const Info = require('./Info');

test('it throws', () => {
    expect(() => Info()).toThrow();
});

test('it works', () => {
    expect(Info({
        part: {
            modifier: {
                atRules: {},
                markup: '<div class="block block--modifier">â€¦</div>',
                name: 'modifier',
            },
        },
        type: 'modifier',
        componentName: 'block',
    })).toMatchSnapshot();
});
