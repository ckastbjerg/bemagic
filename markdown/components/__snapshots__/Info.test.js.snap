exports[`test it works 1`] = `
"
        
        <a name=modifier></a>
        #### Modifier
    
        
            The block component makes use of
            [\`modifiers\`](../CONVENTIONS.md#modifiers) to alter its appearance
        .

        \`\`\`html
        <div class=\"block block--modifier\">â€¦</div>
        \`\`\`

        #####All component \`modifier\`

        
        | Name | Description |
        |:-----|:-----|
        | **\`modifier\`** | N/A |

    
        <sub>See the [Component Overview](#component-overview) for more details</sub>
    "
`;
